deployment_scripts/deploy_linux.sh

```diff
@@ -13,25 +46,18 @@ fi
 
 # We can set main paths after we know the home dir
 ISLAND_PATH="$monkey_home/monkey/monkey_island"
-MONKEY_COMMON_PATH="$monkey_home/monkey/common/"
 MONGO_PATH="$ISLAND_PATH/bin/mongodb"
 ISLAND_BINARIES_PATH="$ISLAND_PATH/cc/binaries"
 INFECTION_MONKEY_DIR="$monkey_home/monkey/infection_monkey"
 MONKEY_BIN_DIR="$INFECTION_MONKEY_DIR/bin"
 
-handle_error() {
-  echo "Fix the errors above and rerun the script"
+if is_root; then
+  echo "Please don't run this script as root"
   exit 1
-}
-
-log_message() {
-  echo -e "\n\n-------------------------------------------"
-  echo -e "DEPLOYMENT SCRIPT: $1"
-  echo -e "-------------------------------------------\n"
-}
```

Comment:
I preffered you left this in the script. It's really hard to pin point what the script did and where one command's output finished and another command's output began without this